.github/workflows/integration_tests.yml

```diff
@@ -60,18 +62,27 @@ jobs:
         if: github.event_name == 'schedule' || github.event.inputs.use_expanded_matrix == '1'
         run: |
           echo "EXPANDED_MATRIX_PARAM=-e=1" >> $GITHUB_ENV
+          echo "::warning ::Expanded test matrix enabled."
```

Comment:
Tip: I used to do this, too, but the fact that the `Use expanded matrix` step is grayed out during the GHA jobs steps, or not grayed out depending on the `if` clause above, also tells the story of the status of the expanded matrix.

Reply:
I removed it from this workflow. I would like to log a few things but we'll take some time to do it thoughtfully.